packages/server/src/plugin/landingPage/default/types.ts

```diff
@@ -0,0 +1,136 @@
+export interface ApolloServerPluginLandingPageDefaultBaseOptions {
```

Comment:
Hmm, not sure what happened here — will revert.

Reply:
Seems to be an oddity of how GH is displaying this — it's not in the squash commit.

Reply:
That's bizarre—nice catch though!